Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicy/MSFT_AADAuthenticationMethodPolicy.schema.mof

```diff
@@ -6,7 +6,6 @@ class MSFT_MicrosoftGraphRegistrationEnforcement
 [ClassVersion("1.0.0")]
 class MSFT_MicrosoftGraphAuthenticationMethodsRegistrationCampaign
 {
-    [Write, Description("Users and groups of users that are excluded from being prompted to set up the authentication method."), EmbeddedInstance("MSFT_MicrosoftGraphExcludeTarget")] String ExcludeTargets[];
```

Comment:
This is a breaking change. Instead what we should do is keep the parameter in the schema and method definitions, remove it from the GET-TargetResource's return, and print warnings in the Get and Set method if the parameter is being used to let users know if has been deprecated.

Reply:
This was a suggestion of Yorick. I need an example how to hadle such situation. Removed the Fix for now

Reply:
Removing the parameter is indeed a breaking change. However adding the missing class (as you did in PR #3605) also fixes the issue 😉